### DISCUSSION TITLE
openSUSE guest, graphical display randomly freezes; openSUSE is still running though

### DISCUSSION THREAD

**Discussion by antonkratz** [utmapp/UTM#6228] [Linux]
Host: Macbook Pro M2
Guest: openSUSE LEAP 15.5
UTM: 4.4.4 (92), virtio-ramfb

Sometimes, at random moments, maybe 2-3 times per day, the graphical display completely freezes. I cannot move the mouse pointer or anything. The graphical display is completely unresponsive and I cannot do anything, keypresses... mouse... seems to have no effect. This also happens at random moments with an Ubuntu guest btw).

However, I also have a second window open which is a text terminal into this openSUSE guest, and I can still log in and work, halt the machine etc... I cannot restart the X server though, if I attempt the terminal is just unresponsive until I Ctrl-C again.

This makes it difficult to be working productively.

Others describe the same issue: https://www.reddit.com/r/Ubuntu/comments/13bk8fv/utm_virtualisation_of_ubuntu_2004_randomly/

**Comment by kmatheussen** [comment]
Very common problem, lots of threads and issues on this topic. Doesn't
seem to happen in fedora 39 though.
